### PR TITLE
Bits & Bobs

### PR DESCRIPTION
**Pull request by farridav**
- Tidy up list view and put into card
- Ensure style is consistent with detail view
- convert all boxes to cards
- ensure we use card-outline with all card-primarys
- fix double row in password change form
- Jazz up login page
- Delete a bunch more unused files :)
- Updated test app to match polls app, easier dev
- reversing custom url

### CODE REVIEW

File: jazzmin/templatetags/jazzmin.py [hunk 1]
```diff
@@ -192,4 +192,5 @@ def debug(value):
     """
     Add in a breakpoint here and use filter in templates for debugging ;)
     """
+    import ipdb; ipdb.set_trace()
```
Comment (TysonRV) [inline jazzmin/templatetags/jazzmin.py:195]: Ha! It's back! Needs the #NOQA label or will flake out

File: jazzmin/templates/registration/logged_out.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <title>{{ jazzmin_settings.site_title }} | {% trans 'Log in again' %}</title>
     <!-- Tell the browser to be responsive to screen width -->
     <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
-    <!-- Bootstrap 3.3.6 -->
+    <!-- Bootstrap 4.5.6 -->
```
Comment (TysonRV) [inline jazzmin/templates/registration/logged_out.html:14]: 4.5.0

File: jazzmin/templates/admin/login.html [hunk 3]
```diff
@@ -1,101 +1,115 @@
...
+                            <label for="remember">
+                                {% trans 'Remember me' %}
+                            </label>
+                        </div>
+                    </div>
+                    <!-- /.col -->
+                    <div class="col-4">
+                        <button type="submit" class="btn btn-primary btn-block">{% trans "Log in" %}</button>
+                    </div>
+                    <!-- /.col -->
```
Comment (TysonRV) [inline jazzmin/templates/admin/login.html:96]: I would keep removing this comments...

File: jazzmin/templates/admin/login.html [hunk 4]
```diff
@@ -1,101 +1,115 @@
...
+    <link rel="stylesheet" href="{% static "jazzmin/css/django.css" %}">
 
-    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-    <script src="//oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-    <script src="//oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
+    <!-- Google Font: Source Sans Pro -->
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700" rel="stylesheet">
```
Comment (TysonRV) [inline jazzmin/templates/admin/login.html:24]: Mr Cheese approves

File: jazzmin/templates/admin/date_hierarchy.html [hunk 5]
```diff
@@ -1,6 +1,5 @@
 {% if show %}
-<div class="col-md-12">
-    <div class="btn-group">
+    <div class="btn-group date-heirarchy text-sm">
```
Comment (TysonRV) [inline jazzmin/templates/admin/date_hierarchy.html:2]: Typo `date-hierarchy`

File: jazzmin/templates/admin/change_form.html [hunk 6]
```diff
@@ -51,24 +45,27 @@
...
-            <div class="col-8">
-                <div class="card card-primary card-outline card-outline-tabs">
-                    <div class="card-header p-0 border-bottom-0">
+            <div class="col-10">
+                <div class="card card-primary card-outline">
+                    <div class="card-header">
+                        <div class="card-title">
+                            {{title}}{% block pretitle %}{% endblock %}
+                        </div>
+                    </div>
```
Comment (TysonRV) [inline jazzmin/templates/admin/change_form.html:54]: I thought it was kind of redundant, but it does look better indeed.

File: jazzmin/templates/admin/change_form.html [hunk 7]
```diff
@@ -51,24 +45,27 @@
...
 
         {% block field_sets %}
-            <div class="col-8">
-                <div class="card card-primary card-outline card-outline-tabs">
-                    <div class="card-header p-0 border-bottom-0">
+            <div class="col-10">
+                <div class="card card-primary card-outline">
+                    <div class="card-header">
+                        <div class="card-title">
+                            {{title}}{% block pretitle %}{% endblock %}
```
Comment (TysonRV) [inline jazzmin/templates/admin/change_form.html:52]: `{{ title }}`

File: jazzmin/static/jazzmin/css/django.css [hunk 8]
```diff
@@ -92,3 +92,7 @@ table.dataTable thead .sorting::after, table.dataTable thead .sorting_asc::after
 .user-profile {
     font-size: 2.4em;
 }
+
+.date-heirarchy {
```
Comment (TysonRV) [inline jazzmin/static/jazzmin/css/django.css:96]: Oh, I see 😄 That explains why it was working

File: jazzmin/templates/admin/change_form.html [hunk 9]
```diff
@@ -86,16 +83,26 @@
...
-
         {% block after_field_sets %}{% endblock %}
 
         {% block inline_field_sets %}{% endblock %}
 
         {% block after_related_objects %}{% endblock %}
 
         {% block submit_buttons_bottom %}
-        <div class="col-4">
+        <div class="col-2">
```
Comment (TysonRV) [inline jazzmin/templates/admin/change_form.html:93]: I think this is too short.. Do you want to use smaller buttons? TBH, it crossed my mind a couple of times as smaller looks also a bit jazzier. Big buttons are manky 😄

**Comment by farridav** [comment]
@TysonRV all updated !

**Comment by TysonRV** [comment]
Nice one! :shipit:

**Comment by TysonRV** [comment]
Test app looks really good as well 👍

**Pull request merged**